README.md

```diff
@@ -59,6 +59,9 @@ Output:
 - [Developer Guide](docs/developer.md)
 - [Operator Design](docs/developer.md)
 
+## Telemetry data collection
+
+The telemetry data collection feature is enabled by default. Red Hat Developer Hub sends telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin. To disable this and to learn what data is being collected, see https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-rhdh-telemetry_admin-rhdh
```

Comment:
Since this repo may at some point become the upstream Backstage operator, I'm not sure we want to mention RHDH in the root readme. WDYT @rm3l  @gazarenkov ? Are we still planning to keep this project "whitelabelled" to eventually move upstream?

Reply:
I would not like we have direct link to RedHat documentation here.
Let's host information of how to disable/enable telemetry plugin here, on Operator repo (see comment below).

Reply:
+1.